Makefile

```diff
@@ -93,17 +96,20 @@ e2e-test:
 
 .PHONY: e2e-bootstrap
 e2e-bootstrap: install-helm
-	docker pull $(IMAGE_TAG) || make container-all push-manifest
 ifdef TEST_WINDOWS
-	helm install csi-driver-smb charts/latest/csi-driver-smb --namespace kube-system --wait --timeout=15m -v=5 --debug \
-		${E2E_HELM_OPTIONS} \
+	docker pull $(WIN_IMAGE_TAG) || make container-all push-manifest	
+	sudo /usr/local/bin/helm upgrade csi-driver-smb charts/latest/csi-driver-smb --namespace kube-system --wait --timeout=15m -v=5 --debug --install \
+		${E2E_WIN_HELM_OPTIONS} \
 		--set windows.enabled=true \
 		--set linux.enabled=false \
 		--set controller.replicas=1 \
 		--set controller.logLevel=6 \
-		--set node.logLevel=6
+		--set node.logLevel=6 \
+		--kubeconfig ~/.kube/config

```

Comment:
I think `~/.kube/config` is the default `--kubeconfig` setting, not necessary to specify here

Reply:
yeah, I thought so. However if I don't specify here, helm fails to connect to. the cluster.

No idea why... Can we keep it here?

Reply:
yeah, also removed --kubeconfig, it seems that it causes prow tests to fail